### ISSUE TITLE
Training steps of ft model

### ISSUE THREAD

**Issue by fishssh** [ruoyuwang1995nya/pfd-kit#73]
The training steps of ft model should be adaptable based on the size of the training dataset.

**Comment by ruoyuwang1995nya** [comment]
This might be task harder than it seems... But as a stop-gap solution, you can try out the recursive fine-tuning in the newest update!  In this mode, a custom probabilitic ratio can be assigned to the new training data sampled in each iteration, and training from the fine-tuned model from the last iteration. By this method the issue of ever increasing training set can be effectively tackled.  You can  also specifiy training parameters (e.g., training steps, learning rate, etc.) of the subsequent fine-tune steps seperate from the initial fine-tune parameter given in the training script. @fishssh
